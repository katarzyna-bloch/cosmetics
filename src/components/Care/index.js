import PageWrapper from '../PageWrapper'

const Care = () => (
  <PageWrapper>
    <p>PielÄ™gnacja</p>
  </PageWrapper>
)

export default Care
