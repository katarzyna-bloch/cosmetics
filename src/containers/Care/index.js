import React, { useEffect } from 'react'
import { connect } from 'react-redux'

import PageWrapper from '../../components/PageWrapper'
import ContentHeader from '../../components/ContentHeader'
import ProductsList from '../../components/ProductsList'
import { requestCareProducts } from '../../redux/actions'

export const Care = ({ fetchProducts, products }) => {
  useEffect(() => {
    fetchProducts()
  }, [])

  return (
    <PageWrapper>
      <ContentHeader>Aby uzyskać zdrowy wygląd cery, należy przestrzegać codzienną pielęgnację twarzy</ContentHeader>
      <ProductsList products={products} />
    </PageWrapper>
  )
}

const mapStateToProps = ({ careProducts }) => ({
  products: careProducts,
})

const mapDispatchToProps = dispatch => ({
  fetchProducts: () => dispatch(requestCareProducts()),
})

export default connect(mapStateToProps, mapDispatchToProps)(Care)
