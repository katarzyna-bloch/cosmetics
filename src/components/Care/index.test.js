import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import Care from '.'

it('should render correctly', () => {
  let wrapped = shallow(<Care />)
  expect(toJson(wrapped)).toMatchSnapshot()
})
