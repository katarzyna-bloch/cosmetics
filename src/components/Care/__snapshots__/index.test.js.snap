// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<styled.div>
  <p>
    PielÄ™gnacja
  </p>
</styled.div>
`;
